src/components/FileTreeNode/index.spec.tsx

```diff
@@ -403,6 +445,57 @@ describe(__filename, () => {
     );
   });
 
+  it('adds a comment icon for a file', () => {
+    const path = 'scripts/background.js';
+
+    const root = renderWithComments({
+      pathsWithComments: [path],
+      ...getTreefoldRenderProps({ id: path }),
+    });
+
+    const icons = root.find(`.${styles.nodeIcons}`).find(FontAwesomeIcon);
+    expect(icons).toHaveLength(1);
+
+    expect(icons.at(0).key()).toEqual('comments-icon');
+  });
+
+  it('adds a comment icon for a directory', () => {
+    const dir = 'scripts';
+    const path = `${dir}/background.js`;
+
+    const root = renderWithComments({
+      pathsWithComments: [path],
+      ...getTreefoldRenderProps({ id: dir }),
+    });
+
+    const icons = root.find(`.${styles.nodeIcons}`).find(FontAwesomeIcon);
+    expect(icons).toHaveLength(1);
+
+    expect(icons.at(0).key()).toEqual('comments-icon');
+  });
+
+  it('adds a linter and comment icon for a file', () => {
```

Comment:
Should there also be a test for "adds a linter and comment icon for a folder", or is that unnecessary?